common/source/docs/common-lua-scripts.rst

```diff
@@ -18,9 +18,10 @@ While scripting normally requires at least some LUA programming and editing, the
 Getting Started
 ===============
 
-- Ensure your autopilot has at least 2 MB of flash and 80 kB of memory.  High powered autopilots like the CubePilot Cube Orange and HolyBro Durandal will certainly work well but check the specifications of your :ref:`autopilot <common-autopilots>`. Scripting is not available in F4 based autopilots. Autopilots must have an SD card to store the script unless the user builds his own firmware and embeds it directly in the firmware (see :ref:`common-oem-customizations`).
-- Set :ref:`SCR_ENABLE <SCR_ENABLE>` to 1 to enable scripting (refresh or reboot to see all ``SCR_`` parameters).
-- Upload scripts (files with extension .lua) to the autopilot's SD card's ``APM/scripts`` folder.
+- Ensure your autopilot has at least 2 MB of flash and 80 kB of memory.  Higher powered autopilots like the CubePilot Cube Orange and HolyBro Durandal will certainly work well but check the specifications of your :ref:`autopilot <common-autopilots>`. Scripting is not available on 1 MB autopilots.
```

Comment:
"CubePilot Cube Orange and HolyBro Durandal"
I realize they are partners, but do we need to shill them in the docs? 

I remember working to change Pixhawk to autopilot..

Reply:
historical, but I can change